website/docs/docs/deploy/slim-ci-jobs.md

```diff
@@ -1,134 +1,67 @@
-
-### Configure CI for a job
-
-If you want dbt Cloud to run the job whenever a pull request or commit is made, you can set up continuous integration (CI) for the job. 
-
-To set up CI:
-
-1. Create a new job or edit an existing job to open the settings page.
-2. Navigate to the **Triggers** section.
-3. Select **Continuous Integration (CI)**. 
-4. Select **Run on Pull Requests?** as shown in the following image.
-
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/ci-tab.png" title="Configuring continuous integration for a dbt Cloud Job"/>
-
-
-### GitHub pull request example
-
-The green checkmark means the dbt builds and tests were successful. The _Details_ link shown here will navigate you to the relevant CI run in dbt Cloud.
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/09c886f-Screen_Shot_2019-02-08_at_4.54.41_PM.png" title="GitHub pull request example"/>
-
-### GitLab pull request example
-
-The green checkmark means the dbt builds and tests were successful. Clicking the dbt Cloud pop up will navigate you to the relevant CI run in dbt Cloud.
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/GitLab-Pipeline.png" title="GitLab pull request"/>
-
-### Azure DevOps pull request example
-
-The green checkmark means the dbt builds and tests were successful. Clicking on the dbt Cloud section navigates you to the relevant CI run in dbt Cloud.
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/ADO CI Check.png" title="Azure DevOps pull request"/>
-
-## Configuring a Slim CI job
-
-Slim CI offers an alternative to running and testing all models in your project, and instead runs and tests only what's necessary based on the changes you've made. Slim CI can decrease the time it takes by running and testing only modified models, which can also reduce unnecessary resource usage on your warehouse/data platform.
-
-A Slim CI job:
-
-- Is triggered by a pull request.
-- Defers to a production job.
-- Includes a command with a `--select state:modified+` selector, which dbt uses to build only new or changed models and their downstream dependents. 
-
-dbt then identifies the models that need to be run and tested using a state comparison to the production job that you've selected.
+    For your dedicated Slim CI job, dbt Labs recommends you set up a separate dbt Cloud environment that's connected to a staging database. This ensures that you have good isolation on where your CI schemas build. Having a separate environment also allows you to set a [custom branch](/faqs/environments/custom-branch-settings) if you want to trigger CI checks off PRs to a pre-release branch.
```

Comment:
i didn't really understand what is meant by "trigger CI checks off PRs to a pre-release branch" meant. can you tell me more? 🙏